pkg/cli/environment.go

```diff
@@ -26,14 +26,21 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/casimir/xdg-go"
 	"github.com/spf13/pflag"
-	"k8s.io/client-go/util/homedir"
 
 	"k8s.io/helm/pkg/helmpath"
 )
 
-// defaultHelmHome is the default HELM_HOME.
-var defaultHelmHome = filepath.Join(homedir.HomeDir(), ".helm")
+// The default value for HELM_HOME.
+//
+// If $XDG_CONFIG_DIR is set, HELM_HOME uses $XDG_CONFIG_DIR/helm as its home directory.
+//
+// Otherwise, the default for HELM_HOME depends on the value of runtime.GOOS:
+//  - windows: %APPDATA%\helm
+//  - darwin:  ~/Library/Preferences/helm
+//  - linux:   ~/.config/helm
+var defaultHelmHome = filepath.Join(xdg.ConfigHome(), "helm")
```

Comment:
This has the same problem as what was discussed in https://github.com/helm/helm/pull/5166#pullrequestreview-194324802. It's resolved by using
```go
var defaultHelmHome = xdg.App{}.ConfigPath("helm")
```

Reply:
Thanks for catching that!